73063.typ

```diff
@@ -472,9 +481,9 @@
 		}
 		= Version 1 #h(1fr) #include "signature.typ"
 		#colbreak()
-		= Operating Checklists #h(1fr) N73146
+		= Operating Checklists #h(1fr) N73063
 		#columns(2)[
-			#set text(8.1pt)
+			#set text(8.3pt)
```

Comment:
Removing "Tach time ... RECORD" from the Shutdown checklist let us increase the font size.

Reply:
Added back, so font size change undone.